.github/workflows/e2e.yml

```diff
@@ -0,0 +1,35 @@
+name: End-to-end Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```

Comment:
We'll want this to run on every PR so the below might be appropriate:
```
pull_request:
  branches:
    - main
```

Reply:
Fixed!